content/en/platform/corda/5.0/application-networks/creating/mgm/register.md

```diff
@@ -152,4 +153,20 @@ Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -
 ```
 {{% /tab %}}
 {{< /tabs >}}
-If successful, you should see the `APPROVED` registration status.
+
+The returned status of the registration request will be one of the following options:
+
+* `APPROVED`: The MGM was onboarded successfully.
+* `NEW`: The member’s Corda cluster has accepted and persisted the registration request submitted via the REST API but is yet to process it.
+* `INVALID`: The member’s Corda cluster processed the registration request submitted via the API and determined the
+input to be invalid and did not attempt to send the request to join the network to the MGM’s Corda cluster.
+* `SENT_TO_MGM`: The member’s Corda cluster processed the registration request submitted via the API and found the input
+to be valid so the request to join the network was forwarded to the MGM’s Corda cluster.
+* `DECLINED`: The MGM has received the registration request and rejected the request either automatically because of
+failed validation or manually by the Network Operator.
+* `PENDING_AUTO_APPROVAL`: The MGM has received the registration request and all automated validations were successful.
+The only remaining step is for the MGM to complete the approval which is an automated process rather than approval
+requiring manual involvement of the Network Operator before the member’s registration request takes effect.
+* `PENDING_AUTO_VERIFICATION`: The MGM has received the registration request and all automated validations were successful.
```

Comment:
Ah, I blindly gave a description here assuming it was the `PENDING_MANUAL_APPROVAL` status. I'm not sure what `PENDING_AUTO_VERIFICATION` is so I need to check that.

Reply:
Maybe that was supposed to be `PENDING_MEMBER_VERIFICATION` - which is described as "A registration request is in this status when the MGM has completed initial validation of the registration request and is now waiting for the registering member to respond to a P2P message in order to validate the provided session initiation key and P2P endpoint."

Reply:
Changed `PENDING_AUTO_VERIFICATION` to `PENDING_MEMBER_VERIFICATION`.